tools/bqtag/README.md

```diff
@@ -0,0 +1,374 @@
+        ]
+    },
+    "type": "STRING"
+    },
+    {
+    "description": "Customer Name",
+    "mode": "REQUIRED",
+    "name": "customer_name",
+    "policyTags": {
+        "names": [
+        "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/7755219766687077882"
+        ]
+    },
+    "type": "STRING"
+    },
+    {
+    "description": "Email",
+    "mode": "REQUIRED",
+    "name": "email_address",
+    "policyTags": {
+        "names": [
+        "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/7755219766687077882"
+        ]
+    },
+    "type": "STRING"
+    },
+    {
+    "description": "Phone Number",
+    "mode": "NULLABLE",
+    "name": "phone",
+    "policyTags": {
+        "names": [
+        "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/461212619990859315"
+        ]
+    },
+    "type": "STRING"
+    },
+    {
+    "fields": [
+        {
+        "mode": "NULLABLE",
+        "name": "street",
+        "policyTags": {
+            "names": [
+            "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/461212619990859315"
+            ]
+        },
+        "type": "STRING"
+        },
+        {
+        "mode": "NULLABLE",
+        "name": "city",
+        "policyTags": {
+            "names": [
+            "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/4808767303335996791"
+            ]
+        },
+        "type": "STRING"
+        },
+        {
+        "mode": "NULLABLE",
+        "name": "state",
+        "policyTags": {
+            "names": [
+            "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/4808767303335996791"
+            ]
+        },
+        "type": "STRING"
+        },
+        {
+        "mode": "NULLABLE",
+        "name": "pincode",
+        "policyTags": {
+            "names": [
+            "projects/<project_id>/locations/us/taxonomies/355XXXXXXXXXXX5604/policyTags/7755219766687077882"
+            ]
+        },
+        "type": "STRING"
+        }
+    ],
+    "mode": "NULLABLE",
+    "name": "address",
+    "type": "RECORD"
+    }
+]
+```
+
+This function returns the schema of the table created as a dictionary. If there was an error or exception, a `{}` is returned. 
+
+## Create an Authorized View using Tags
+
+`create_view()` function can create the authorized BQ view by only including columns that are tagged with a particular value. While creating the view, this function takes care of the format of original table structure meaning if original table has nested and repeated columns then the created view would also have nested and repeated columns. Dataset and BigQuery Project, defined at the time of initialization of the object is used as the dataset and project for the new view. This function takes the following attributes:
```

Comment:
The `create_view()` function...

Reply:
Included the suggestion

Reply:
Changed to:

The `create_view()` function can create